website/blog/2024-04-10-roadmap-q2.md

```diff
@@ -0,0 +1,96 @@
+---
+slug: 2024-q2-roadmap
+title: 2024 Q2 Roadmap update
+authors:
+  name: Sam Richards
+  title: MACI team lead
+  url: https://x.com/samonchain
+  image_url: https://avatars.githubusercontent.com/u/8097623?v=4
+tags: [roadmap, rpgf, qf]
+---
+
+Greetings anon,
+
+Glad to have you here! We have a few MACI roadmap updates to share.
+
+Before we hop into our Q2 plans, let's take a look at what we did in Q1:
+
+## Q1 in review
+
+In Q1 we aimed at [a few major goals](https://github.com/privacy-scaling-explorations/maci/discussions/859#discussioncomment-7849385) that all tied together:
+
+### ✅ Release MACI v1.2
+
+We completed a significant refactor to simplify the codebase and improve developer experience. After an internal audit and revamped documentation, we released a new MACI version - see our [MACI v1.2 release post](/blog/maci-v1-2-0-release) to read up on the details of that upgrade.
+
+### ✅ Support clr.fund upgrade
+
+[clr.fund](https://clr.fund/#/), a quadratic funding (QF) project which has distributed millions of dollars to Ethereum public goods, is the longest-running production application built on MACI (since 2020!). Thanks primarily to the hard work of their core dev [yuetloo](https://github.com/yuetloo), clr.fund successfully upgraded from 0.x to our fresh v1.2 release. Nicely done!
+
+### ✅ Support ETH Latam QF round
+
+We teamed up with [ETH Latam](https://ethlatam.org/) to support their efforts to run a QF round using the clr.fund stack (running on MACI v1.2). Over 60 conference attendees participated (using [Zupass](https://zupass.org/) tickets as the gatekeeper to the faucet and round) to distribute over 30,000 DAI to public good projects across Central America and the Carribean - [view the round results](https://qf.ethlatam.org/#/leaderboards/0x86F33909474c0dEf2Cb7F93d2eE0B8aF26112BF6/networks/optimism) and learn more in [ETH Kipu's ETH Latam recap post](https://mirror.xyz/ethlatam.eth/OoDqW3Omy8NbOGosdDQ8XUp_fZjP4sf_s4VHkaPWZXM)!
+
+### ✅ Support ETH Colombia QF round
+
+Kudos to the ETHColombia team, who ran a [QF round for their community](https://www.ethcolombia.org/quadratic-funding-ethco-2024-q1) on the [#RumboaETHLatam](https://twitter.com/hashtag/RumboaETHLatam) - [view the round results here](https://qf.ethcolombia.org/#/leaderboards/0xa73Ec044b47186646D84D614b8a194dA3bE00260/networks/optimism)!
```

Comment:
I would change it to "The ETH Colombia team ran a QF round for their community [insert link]. Would leave out the kudos since it is a challenging time for them and would choose to just write it in a tone that is stating that this happened